immersion/kubeflow_pipelines/walkthrough/solutions/lab-01_vertex.ipynb

```diff
@@ -0,0 +1,880 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    JOB_NAME = f"JOB_VERTEX_TUNNING_{TIMESTAMP}"
</pre><p>typo, should be 'TUNING' (and below elsewhere in the lab, search for "tunning")</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/mlops-on-gcp/pull/115/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='39'/>

Reply:
<p>Done!</p>